src/desktop/lib/global_react_modules.js

```diff
@@ -1,153 +1,19 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
-import { Artwork } from 'reaction/Components/Artwork'
-import { ArtworkGrid } from 'reaction/Components/ArtworkGrid'
-import { Fillwidth } from 'reaction/Components/Artwork/Fillwidth'
-import { ServerStyleSheet } from 'styled-components'
-import { data as sd } from 'sharify'
-import { renderToString } from 'react-dom/server'
-import { isFunction, uniqueId } from 'lodash'
+import { Artwork as _Artwork } from '@artsy/reaction/dist/Components/Artwork'
+import { ArtworkGrid as _ArtworkGrid } from '@artsy/reaction/dist/Components/ArtworkGrid'
+import { Fillwidth as _Fillwidth } from '@artsy/reaction/dist/Components/Artwork/Fillwidth'
 
-export class ReactionRenderer {
-  id
-  mode
-  type
-  res
-  props
+export const Artwork = props => (
+  <_Artwork artwork={artwork} {...props} useRelay={false} />
+)
+export const ArtworkGrid = props => (
+  <_ArtworkGrid artworks={artworks} {...props} useRelay={false} />
+)
+export const Fillwidth = props => (
+  <_Fillwidth artworks={artworks} {...props} useRelay={false} />
+)
```

Comment:
This file will be cleaned up in a subsequent PR, using it as a bit of a testbed for now